content/source/404.html.erb

```diff
@@ -0,0 +1,18 @@
+---
+layout: "inner"
+page_title: "Not Found"
+noindex: true
+description: |-
+  Page not found!
+---
+
+<h1>Page Not Found</h1>
+
+<p>Sorry, the page you tried to visit does not exist. This could be our fault, and if so we will fix that up right away.</p>
+
+<p>Please go back to get back on track.</p>
+
+<script>
+  // Fire a custom event to Segment to surface paths that are causing 404s
+  window && window.analytics && window.analytics.track('404 Response')
```

Comment:
Segment captures the relevant info like referrer, path etc. that we want here. This event name (`404 Response`) is consistent with what we're using with our other web properties.